### DISCUSSION TITLE
Wezterm like Quick Select

### DISCUSSION THREAD

**Discussion by TheTrio** [kovidgoyal/kitty#5695] [Ideas]
Hey! So I've been using kitty for a long time now but I recently tried Wezterm. After a week or so, I switched back to Kitty due to the easier configuration options but having used Wezterm, I was increasing relying on its [Quick select mode](https://wezfurlong.org/wezterm/quickselect.html) to copy short pieces of text.

Is there a way to emulate that behavior in Kitty?

Thanks!

**Comment by kovidgoyal** [comment]
https://sw.kovidgoyal.net/kitty/kittens/hints/

**Reply by TheTrio** [reply]
Thanks!
